java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1908,6 +1908,17 @@ public ColumnVector endsWith(Scalar pattern) {
     }
   }
 
+  /**
+   * Removes whitespace from the beginning and end of a string.
+   * @return A new java column vector containing the stripped strings.
+   */
+  public ColumnVector strip() {
+    assert type == DType.STRING : "column type must be a String";
+    try (DevicePrediction prediction = new DevicePrediction(predictSizeFor(DType.INT32), "strip")) {
```

Comment:
This is likely to go away soon, so it is just a nit, but the output of this will likely be close to the original size instead of the size of an int array the same length.